src/executor.js

```diff
@@ -4,7 +4,10 @@ var factory = require("./response-factory"),
     Test = require("./test"),
     screenshot = require("./screenshot"),
     stepRegistry = require("./step-registry"),
-    hookRegistry = require("./hook-registry");
+    hookRegistry = require("./hook-registry"),
+    screenshotFactory = require("./screenshot-factory"),
```

Comment:
why is the message module called `customMessageRegistry` and screenshot `Factory`? These are similar, so makes sense to keep naming consistent